docs/private-unfurls.md

```diff
@@ -0,0 +1,43 @@
+# Private unfurls
+
+Below are mockups for potential flows that need to be implemented before we can make private unfurls generally available.
+
+## Current flow
+
+Show Rich preview
+![Rich preview demo](https://d2mxuefqeaa7sj.cloudfront.net/s_4ACE98FACF4997A859FBDCF23940D470F9FBABA31346EEED9CA69C21A513E7A0_1522849637496_rich+preview.gif)
+
+Dismiss
+![Dismiss demo](https://d2mxuefqeaa7sj.cloudfront.net/s_4ACE98FACF4997A859FBDCF23940D470F9FBABA31346EEED9CA69C21A513E7A0_1522849647345_dismiss.gif)
+
+## Potential flow
+
+`1`. Initial prompt
```

Comment:
I feel like maybe this initial prompt should have a 'Don't ask me again' button in addition to 'show rich preview' and 'dismiss. That would then trigger the 'Getting too many of these prompts?' flow